eng/Packages.Data.props

```diff
@@ -108,8 +108,8 @@
   <ItemGroup Condition="'$(IsClientLibrary)' == 'true' and $(MSBuildProjectName.StartsWith('Microsoft.'))">
     <PackageReference Update="CloudNative.CloudEvents" Version="2.0.0" />
     <PackageReference Update="CloudNative.CloudEvents.SystemTextJson" Version="2.0.0" />
-    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.27" />
-    <PackageReference Update="Microsoft.Azure.WebJobs.Sources" Version="3.0.19" />
+    <PackageReference Update="Microsoft.Azure.WebJobs" Version="3.0.30-11865" />
```

Comment:
This is a temporary reference to a preview package from our myget feed to allow this PR to compile. Before checkin, we'll reference a non-prerelease package from Nuget.